ChangeCalculationTests/ChangeCalculatorTest.cs

```diff
@@ -1,11 +1,100 @@
+using ChangeCalculation;
+
 namespace ChangeCalculationTests
 {
     public class ChangeCalculatorTest
```

Comment:
Check this case:
You have the following coins: 50 10 5 2 2 2. And you ask for 61. Can you return that amount?